### ISSUE TITLE
List numbering does not reset in recent versions of Firefox

### ISSUE THREAD

**Issue by JorisDebonnet** [quilljs/quill#3309]
See this Stack Overflow question: https://stackoverflow.com/questions/53420948/strange-browser-behavior-with-counter-identifier-list-item

The way the Quill list-items are numbered does not work in Firefox: the counter resets do not work. The fix is very simple, but I could not find where the css is hosted on Github, otherwise I'd just send a PR. 

The issue is that in Firefox, Quill will show lists like this: 
```
1. Line
   a. Line
   b. Line
2. Line
   c. Line
3. Line
   d. Line
```
In short, the counter-reset rules don't work. The reason (as silly as it is) is that they are named `list-0`, `list-1`. Just name them anything else, and that fixes it. In my styles, I've renamed them to `ql-0`, `ql-1` etc, which fixes the issue.
